infrastructure/config.sh

```diff
@@ -18,8 +18,8 @@
 export aws_region="eu-west-1"
 export dynamo_table_name="terraform-lock"
 export bucket_name="terraform-state-xc3"
-export project="example"
-export domain="testing.example.co"
-export owner_email="admin@example.co"
-export creator_email="admin@example.co"
-export namespace="example"
+export project="yaseen"
```

Comment:
it should be generic please remove your name

Reply:
+1